cmake/UseMultiArch.cmake

```diff
@@ -1,3 +1,5 @@
+#FIXME: You can use https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html instead
```

Comment:
`UseMultiArch.cmake` was copied from Kodi's platform library and I wasn't sure what it actually does. But I know it was very importand.
So when I use GNUInstallDirs `UseMultiArch.cmake` obsolete right?